csun-metalab/nomi
# NAM-478 -- UI: Fix color contrast in all themes

# Description
Used Wave plugin to check for contrast and adjust colors; created new bright themes; changed main vue component loading to nomi icon.
  
# Testing
1. Login and notice the new loading icon.
2. Visit settings and ensure all the themes have the proper color and names.
3. Go through each theme and ensure it meets AA (minimal) wcag contrast standard excluding the navbar on the bottom.
4. Ensure all colors are appealing and correct on each theme.
5. Test all pages to see if the correct colors are applied or none at all.
